tools/packaging/kernel/configs/fragments/whitelist.conf

```diff
@@ -15,3 +15,9 @@ CONFIG_NF_LOG_COMMON
 CONFIG_MANDATORY_FILE_LOCKING
 CONFIG_ARM64_UAO
 CONFIG_VFIO_MDEV_DEVICE
+
+# newly added with v5.19
+CONFIG_SPECULATION_MITIGATIONS
+# discontinued with v5.18
+CONFIG_ND_BLK
+CONFIG_DEV_PAGEMAP_OPS
```

Comment:
So this and all the 5.19 work should probably be already in #4861.

Reply:
Correct